projects/core/src/user/store/effects/payment-methods.effect.ts

```diff
@@ -75,8 +80,27 @@ export class UserPaymentMethodsEffects {
     })
   );
 
+  @Effect({ dispatch: false })
```

Comment:
need a unit test for this. I also noticed that there is no spec file for `payment-method.effects` so would have to create a whole new file and add specs for all other effects as well.